mscore/plugin/api/cursor.cpp

```diff
@@ -161,24 +161,24 @@ bool Cursor::nextMeasure()
 //---------------------------------------------------------
 //   add
 ///   Adds the given element to a score at this cursor's
-///   position.
+///   position. Optional parent element will be used if not null 
 //---------------------------------------------------------
 
-void Cursor::add(Element* wrapped)
+void Cursor::add(Element* wrapped, Element* parent /*= nullptr*/)
```

Comment:
Also, if possible, it would be preferable not to keep these changes in intermediate commits: they are reverted in the same PR anyway, but git history will remember this line as changed twice in these commits.

Reply:
alright, after this next change, i will create a single commit (instead of two) then this will go away